### DISCUSSION TITLE
Multi Mod Management

### DISCUSSION THREAD

**Discussion by NillieZilla** [gameserverapp/Platform#342] [Suggestions]
Could we perhaps add the ability to remove multiple mods within the Dashboard? Perhaps even update multiple mods.

**Comment by Forsakenfx** [comment]
Because it's related, would like to see being able to enable/disable mods (for easier debugging for mod crashes on Ark), being able to import from a list, or export a list of mods.

**Comment by VenomOfSkadi** [comment]
This! When I'm changing up mods or I have to do a mass reinstall to hopefully fix an issue, it would make life o much easier to be able to uninstall all. I love the install IDs button cause I can install all the pasted in IDs, but the remove all would be so helpful as well.

**Comment by mvaessen** [comment]
Removing multiple mods in a single go will be available later today
